data/mswat5.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "Swatanra Mishra",
+  "type": "personal",
+  "bio": "developer developing"
```

Please add the missing comma `,` here! 

```suggestion
  "bio": "developer developing",
```